src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,280 @@
+    Because libp2p was to be used in the Beacon Chain, the project specifications were given a lot more scrutiny.
+    This empowered libp2p maintainers to invest a lot of time into making the specs the best possible.
+    One example of this was **adding a new Noise encryption spec**.
+    An [open question posited in an early wire protocol draft](https://github.com/ethereum/consensus-specs/issues/692) considered whether to use Noise (not yet standardized in libp2p) in lieu of Secio or TLS 1.3 (both supported in libp2p).
+    This created the motivation to [create a spec for and standardize the Noise handshake in libp2p](https://github.com/libp2p/specs/issues/195).
+    Many [key developers of Ethereum 2.0 were also formally added](https://github.com/libp2p/specs/issues/232) as interest group members of the Noise spec per the [libp2p spec process](https://github.com/libp2p/specs/blob/master/00-framework-02-document-header.md#authors-and-interest-group).
+    Noise was eventually [added as formal requirement](https://github.com/ethereum/eth2.0-pm/issues/106) of the networking spec and teams started building new implementations (like [js-libp2p-noise](https://github.com/ethereum/eth2.0-pm/issues/106#issuecomment-558805339).)
+2. **Hardening & formalization of existing libp2p implementations:**
+    GossipSub was selected for use and this resulted in efforts to extensively test it.
+    The [Ethereum Foundation and ConsenSys co-funded a grant](https://blog.ethereum.org/2019/08/26/announcing-ethereum-foundation-and-co-funded-grants) to *"analyze the libp2p gossipsub implementation, ...help refine the networking stack and specification, and advance interoperability efforts."*
+    This effort was led by [Whiteblock](https://medium.com/whiteblock/ethereum-2-0-networking-grant-update-1-6f988dac4c7b) who published their tests and results as a part of their [gossipsub-tests](https://github.com/Whiteblock/gossipsub-tests) and [p2p-tests](https://github.com/whiteblock/p2p-tests).
+    Protocol Labs also started to formalize the GossipSub implementation.
+3. **Analysis of the colloboration:**
+    The effort of integrating libp2p in Eth 2.0 also presented a valuable opportunity to reflect on what was going well in the collobration and areas that Protocol Labs could improve on.
+    In July 2019, a [study of libp2p and Eth2](https://discuss.libp2p.io/t/report-a-study-of-libp2p-and-eth2/229) was shared.
+    The report detailed struggles shared by Ethereum teams in the integration process and provided recommendations on how to overcome these issues.
+
+## Full Speed Ahead (2020-present) 🏎️
+
+The Eth 2.0 team started hosting dedicated "networking" meetings (started in Dec 2019) in addition to implementer calls. These meetings were dedicated to [discussing progress and unblocking libp2p related endeavors](https://github.com/ethereum/eth2.0-pm/issues/124#issuecomment-579740557) (GossipSub paper & peer scoring, Noise, etc.)
```

Comment:
Worth mentioning that the presence of libp2p core team members was continuous and important in these meetings.

Reply:
Added